aws/src/test/java/org/apache/iceberg/aws/TestAwsClientFactories.java

```diff
@@ -34,7 +34,6 @@
 import software.amazon.awssdk.services.kms.KmsClient;
 import software.amazon.awssdk.services.s3.S3Client;
 
-
```

Comment:
Please revert this change. We avoid changes that are unnecessary and could cause merge failures.

Reply:
Thanks, noted and made the changes.